src/OpenLoco/GameCommands/VehiclePickup.cpp

```diff
@@ -0,0 +1,68 @@
+#include "../Interop/Interop.hpp"
+#include "../Management/Expenditures.h"
+#include "../Types.hpp"
+#include "../Vehicles/Vehicle.h"
+#include "GameCommands.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    static loco_global<uint8_t, 0x009C68EA> gGameCommandExpenditureType; // premultiplied by 4
+
+    // 0x0048B15B
+    static void playPickupSound()
```

Comment:
I think it would be better if this took a `const &map_pos3` and we passed it the value from `veh2->x,y,z` that way it is clear where this value is coming from.

Reply:
Agreed. The function itself isn't even that complicated. I just wasn't sure how to adapt it with our prng utility. Any ideas?

Reply:
`static loco_global<Utility::prng, 0x00525E20> _prng;` its used in the IndustryList window. Suspect this is the Ui prng that isn't required to be in sync.

Reply:
volume is edi (-1000), frequency is ebx (20003 -> 24098), eax is the sound id